dev/fattest.simplicity/src/componenttest/annotation/processor/TestServletProcessor.java

```diff
@@ -98,6 +98,10 @@ private static String getQueryPath(TestServlet anno) {
     }
 
     private static Method[] getTestServletMethods(TestServlet anno) {
+        if (!!!(javax.servlet.http.HttpServlet.class.isAssignableFrom(anno.servlet()) ||
+                jakarta.servlet.http.HttpServlet.class.isAssignableFrom(anno.servlet()))) {
+            throw new IllegalArgumentException("For the @TestServlet annotation, servlet() must be a subclass of javax.servlet.http.HttpServlet or jakarta.servlet.http.HttpServlet!");
```

Comment:
Lets also include `anno.toString()` in this error message so that if a FAT has multiple `@TestServlet` annotations they can know which annotation had the bad servlet() value

Reply:
I modified to the error message to be as clear as possible, including the annotation string of course, and provided a user-action.